go.mod

```diff
@@ -35,79 +34,224 @@ require (
 )
 
 require (
-	cloud.google.com/go v0.97.0 // indirect
+	github.com/golangci/golangci-lint v1.49.0
```

Comment:
Is that intentional to have this in separate block?

Reply:
The file has been generated by `go mod tidy`. Probably there is a limit on how many items you can have in one section.